soc/arm/st_stm32/stm32g4/Kconfig.defconfig.series

```diff
@@ -10,4 +10,12 @@ source "soc/arm/st_stm32/stm32g4/Kconfig.defconfig.stm32g4*"
 config SOC_SERIES
 	default "stm32g4"
 
+config DMA_STM32_V2
+	default y
+	depends on DMA_STM32
+
```

Comment:
With the new kconfig thing is this still needed? @FRASTM  @erwango ?

Reply:
no more, because of  the #31163

Reply:
@FRASTM Well something isn't working with out it, any ideas ?

Reply:
Instead of `compatible = "st,stm32-dma"` it should be `compatible = "st,stm32-dma-v2"`

Reply:
@shlomow thanx, fixed